module #2. REST API Basics/rest-api-basic/core/src/main/resources/database/mysql/db_script.sql

```diff
@@ -0,0 +1,68 @@
+-- MySQL Workbench Forward Engineering
```

Comment:
I cannot find any place (configuration) where this script file will be taken for processing. Can you configure it. If it should be executed manually then can you mention it in README.md file